.github/workflows/spelling.yml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: 'Spelling'
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+permissions:
+  contents: read
+concurrency:
+  group: '${{ github.workflow }}-${{ github.ref }}'
+  cancel-in-progress: true
+jobs:
+  typos_check:
+    name: '文A Typos check'
```

Comment:
Some odd character here (renders as a square for me)?

Reply:
It tries to resemble https://commons.wikimedia.org/wiki/File:Translation_icon_cropped.svg

Removing ...

Reply:
BTW Are you running Linux that https://en.wiktionary.org/wiki/%E6%96%87 is not included in your font?